openpype/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -145,13 +170,13 @@ def get_instance_attr_defs(self):
                         "color": "Dilation + default background color",
                         "diffusion": "Dilation + diffusion"
                     },
-                    default="infinite",
+                    default=self.exportPadding,
                     label="Padding"),
             NumberDef("exportDilationDistance",
                       minimum=0,
                       maximum=256,
                       decimals=0,
-                      default=16,
+                      default=self.exportDilationDistance,
```

Comment:
I don't feel like we should update the defaults based on the pre create data at all. The defaults are the defaults. If someone tweaked it in pre create then it's still changed from the default value so I wouldn't bother with tweaking the defaults.

Reply:
Agreed.